backend/src/test/java/com/woowacourse/levellog/fixture/TimeFixture.java

```diff
@@ -0,0 +1,10 @@
+package com.woowacourse.levellog.fixture;
+
+import java.time.LocalDateTime;
+
+public class TimeFixture {
+
+    public static final LocalDateTime FAKE_NOW_TIME = LocalDateTime.now().plusDays(5);
+    public static final LocalDateTime AFTER_FAKE_NOW_TIME = LocalDateTime.now().plusDays(5).plusHours(1);
+    public static final LocalDateTime BEFORE_FAKE_NOW_TIME = LocalDateTime.now().plusDays(5).minusHours(1);
```

Comment:
로마 픽스처 잘 만드셨네요!!
다만 변수 이름이 명확하지 않아서 한번 고려해봤으면 좋겠어요.
1. `FAKE`라는 단어를 꼭 넣어야하나요? 저는 `FAKE`는 전부 빼도 괜찮아보여요.
2. 테스트 코드에서 사용되는 상황에 맞게 이름을 변경해보면 어떨까요?
아래의 느낌으로 변경하면 테스트 코드를 읽기 더 수월할 것 같아요. 😉

|              |현재                       |          (1시간)미래                   |       (1시간)과거                         |
|----------|--------------------------|--------------------------------------|-----------------------------------------|
|   AS-IS  |  `FAKE_NOW_TIME`  |  `AFTER_FAKE_NOW_TIME`  |  `BEFORE_FAKE_NOW_TIME`  |
|   TO-BE  |   `NOW`  |  `FUTURE`  |    `PAST`    |

---

추가로 아래와 같이 바꾸는게 좋아보이네요
```suggestion
    public static final LocalDateTime FAKE_NOW_TIME = LocalDateTime.now().plusDays(5);
    public static final LocalDateTime AFTER_FAKE_NOW_TIME = FAKE_NOW_TIME.plusHours(1);
    public static final LocalDateTime BEFORE_FAKE_NOW_TIME = FAKE_NOW_TIME.minusHours(1);
```

Reply:
FAKE 빼는 거 좋아요👍🏼
단순히 PAST 보다는 테스트 상의 현재 시점(NOW)과 비교했을 때 과거인 걸 알 수 있는 이름이 덜 헷갈릴 거 같아요